backend/dal/Services/Concrete/ProjectService.cs

```diff
@@ -952,7 +956,7 @@ private async System.Threading.Tasks.Task<IEnumerable<NotificationQueue>> Genera
         /// <param name="responses"></param>
         /// <param name="noteChanged"></param>
         /// <returns></returns>
-        private async System.Threading.Tasks.Task SendNotificationsAsync(Project project, int? fromStatusId, IEnumerable<ProjectAgencyResponse> responses, bool noteChanged)
+        private async System.Threading.Tasks.Task SendNotificationsAsync(Project project, int? fromStatusId, IEnumerable<ProjectAgencyResponse> responses, bool noteChanged, bool sendNotification = true)
```

Comment:
It doesn't make sense to have a function called `SendNotificationAsync` which then has an option that stops sending notifications.